contentcuration/contentcuration/frontend/shared/data/serverSync.js

```diff
@@ -200,6 +316,38 @@ function handleMaxRevs(response, userId) {
   return Promise.all(promises);
 }
 
+async function WebsocketSendChanges(changesToSync) {
+  const user = await Session.getSession();
```

Comment:
I think the simplest thing to do here might be to make this parallel to the change sending in `syncChanges`, and here instead of passing in `changesToSync` we do the same `syncActive` blocking, and get the syncable changes to send here in exactly the same way as in `syncChanges`. I see a lot of the logic is already parallel, so I think this will work well.